### ISSUE TITLE
循环神经网络的梯度问题 | Alex_McAvoy

### ISSUE THREAD

**Issue by Alex-McAvoy** [Alex-McAvoy/Gitalk-Comment#872]
https://alex-mcavoy.github.io/artificial-intelligence/deep-learning/cd299b62.html 

 References：  Learning Longer Memory in Recurrent Neural Networks Learning long-term dependencies with gradient descent is difficult RNN 的梯度消失问题 RNN梯度消失与梯度爆炸推导 理论推导RNN梯度消失和梯度爆炸的原因 RNN梯度消失和爆炸的原因   【概述】
